.travis.yml

```diff
@@ -7,8 +7,13 @@ php:
   - nightly
   - hhvm
 
-install:
+
+before_script:
```

Comment:
I looked at the TravisCI doc. They recommend using the "install" phase to install dependencies. Also --dev is deprecated and --prefer-source is not useful in our case since we start from scratch every build. `composer self-update` can also be removed (the doc states that the warning can be safely ignored).

And per Coveralls doc, the mkdir should be in the "script" phase.